addons/l10n_es/data/account_fiscal_position_template_data.xml

```diff
@@ -24,7 +24,12 @@
         </record>
 
         <record id="fp_extra" model="account.fiscal.position.template">
-            <field name="name">Régimen Extracomunitario / Canarias, Ceuta y Melilla</field>
+            <field name="name">Régimen Extracomunitario</field>
+            <field name="chart_template_id" ref="account_chart_template_common"/>
+        </record>
+
+        <record id="fp_canarias_ceuta_melilla" model="account.fiscal.position.template">
+            <field name="name">Régimen Canarias, Ceuta y Melilla</field>
```

Comment:
Is there a way we can do the auto_apply here?  Based on state/zip/...

Reply:
It's not only for Canary Island. Also apply in a "Servicios" and "Bienes" from one country that it's not Spain to another country that is not Spain, but the company that offer it is in Spain. I know that is a special case, but neccesary.

Reply:
@acysos Ok, so there is another special case than just the Canary Islands.  But it is a special case, so even if we put the auto_apply, the user can still choose differently afterwards.  (Do people fill in the province in the addresses in Spain? (then I have Las Palmas, Santa Cruz de Tenerife and Ceuta and Melilla))

Reply:
If I understand, you suggest use the default values for this provinces, similar to USA cloud tax?

Reply:
@jco-odoo The special case is the user to have to change the tax in the invoice lines. Because is imposible to know if it sin't subject by tax localizations (send for other country not spain to another country not spain) or is an export (send from spain to other country). 
But in from Spain to Canary always is not subject, and can be "Bienes" or "Services".

Reply:
About the auto_apply, is only for countries, not for province-state?